commands/team_test.go

```diff
@@ -825,3 +830,84 @@ func (s *MmctlUnitTestSuite) TestModifyTeamsCmd() {
 			printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestRestoreTeamsCmd() {
+	teamName := "team1"
+	teamID := "teamId"
+	cmd := &cobra.Command{}
+
+	s.Run("Restore teams with team not exist in db returns an error", func() {
+		printer.Clean()
+
+		s.client.
+			EXPECT().
+			GetTeamByName(teamName, "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeam(teamName, "").
+			Return(nil, &model.Response{Error: nil}).
+			Times(1)
+
+		err := restoreTeamsCmdF(s.client, cmd, []string{"team1"})
+		s.Require().Nil(err)
+		s.Require().Equal("Unable to find team 'team1'", printer.GetErrorLines()[0])
+	})
+
+	s.Run("Restore team", func() {
+		printer.Clean()
+		mockTeam := model.Team{
+			Id:   teamID,
+			Name: teamName,
+		}
+
+		s.client.
+			EXPECT().
+			RestoreTeam(teamID).
+			Return(&mockTeam, &model.Response{Error: nil}).
+			Times(1)
+
+		s.client.
+			EXPECT().
+			GetTeam(teamName, "").
+			Return(&mockTeam, &model.Response{Error: nil}).
+			Times(1)
```

Comment:
Same mock order comment here

Reply:
Right, I did the fix before merging from master.  Thanks.